### DISCUSSION TITLE
Deploying using Dokku and Good Job external mode

### DISCUSSION THREAD

**Discussion by bradbajuz** [bensheldon/good_job#1222] [Q&A] [answered]
Hello, I've been trying to get Good Job working in external mode for a couple days. I'm deploying to a DigitalOcean droplet using Dokku, which uses Docker containers. When I add: `worker: bundle exec good_job start` to my `Procfile` and try to deploy, the deploy fails with the Puma error: `ERROR: No application configured, nothing to run`.

I'm running Puma in Clustered Mode.

My `Procfile`:
```
web: bundle exec puma -C config/puma.rb
worker: bundle exec good_job start
```

Has anyone else had experiencing deploying using Dokku?

**Comment by bradbajuz** [comment, marked as answer]
Issue had to do with the line separator of the Procfile being CRLF. Changed it to LF and deploy worked.

Bug was reported on [Dokku](https://github.com/dokku/dokku/issues/6490).

**Reply by bensheldon** [reply]
wow, glad you were able to figure that out 🤯
